extensions/amp-ad-network-oblivki-impl/0.1/amp-ad-network-oblivki-impl.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
This is a new file, can you correct the year to `2020`?

```suggestion
 * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
```